resources/helpers.php

```diff
@@ -366,319 +369,344 @@ function zype_get_all_plans() {
+        return $zype_search['term'];
+    }
+    return false;
 }
 
-function zype_sort(){
-  global $zype_sort;
-  if(isset($zype_sort['order']) && $zype_sort['order']){
-    return $zype_sort['order'];
-  }
-  return false;
+function zype_sort()
+{
+    global $zype_sort;
+    if (isset($zype_sort['order']) && $zype_sort['order']) {
+        return $zype_sort['order'];
+    }
+    return false;
 }
 
-function zype_sort_querystring(){
-  $sort = zype_sort();
-  if($sort){
-    return 'sort='.$sort;
-  }
+function zype_sort_querystring()
+{
+    $sort = zype_sort();
+    if ($sort) {
+        return 'sort=' . $sort;
+    }
 }
 
-function zype_search_querystring(){
-  $search = zype_search();
-  if($search){
-    return 'search='.$search;
-  }
+function zype_search_querystring()
+{
+    $search = zype_search();
+    if ($search) {
+        return 'search=' . $search;
+    }
 }
 
-function zype_search_and_sort_querystring(){
-  $search = zype_search_querystring();
-  $sort = zype_sort_querystring();
-
-  $querystring = '';
-
-  if($search || $sort){
-    $querystring = '?';
-    if($search && $sort){
-      $querystring .= $search.'&'.$sort;
-    }
-    elseif ($search){
-      $querystring .= $search;
-    }
-    elseif ($sort){
-      $querystring .= $sort;
+function zype_search_and_sort_querystring()
+{
+    $search = zype_search_querystring();
+    $sort = zype_sort_querystring();
+
+    $querystring = '';
+
+    if ($search || $sort) {
+        $querystring = '?';
+        if ($search && $sort) {
+            $querystring .= $search . '&' . $sort;
+        } elseif ($search) {
+            $querystring .= $search;
+        } elseif ($sort) {
+            $querystring .= $sort;
+        }
     }
-  }
-  return $querystring;
+    return $querystring;
 }
 
-function current_menu_item($classes=[], $item=false){
-  $current_url = trailingslashit(get_site_url(null, filter_var($_SERVER['REQUEST_URI'], FILTER_SANITIZE_STRING)));
-  if(strstr($current_url, $item->url)){
-    $classes[] = 'current-menu-item';
-  }
-  return $classes;
+function current_menu_item($classes = [], $item = false)
+{
+    $current_url = trailingslashit(get_site_url(null, filter_var($_SERVER['REQUEST_URI'], FILTER_SANITIZE_STRING)));
+    if (strstr($current_url, $item->url)) {
+        $classes[] = 'current-menu-item';
+    }
+    return $classes;
 }
+
 add_filter('nav_menu_css_class', 'current_menu_item', 10, 2);
 
-function zype_wp_enqueue_scripts() {
+function zype_wp_enqueue_scripts()
+{
     wp_enqueue_script('jquery');
     wp_enqueue_script('jquery-form');
 }
+
 add_action('wp_enqueue_scripts', 'zype_wp_enqueue_scripts', 99);
 
-function the_share_buttons($url=null){
-    if(!$url){
-        $url = urlencode(site_url().filter_var($_SERVER['REQUEST_URI'], FILTER_SANITIZE_STRING));
-    }
-    else{
+function the_share_buttons($url = null)
+{
+    if (!$url) {
+        $url = urlencode(site_url() . filter_var($_SERVER['REQUEST_URI'], FILTER_SANITIZE_STRING));
```

Comment:
@Osoro Same validation stuff here, also for line [688](https://github.com/zype/zype-wordpress-plugin/pull/38/files?utf8=%E2%9C%93&diff=split&w=1#diff-7cd6c410c332ccce15ebb07a258fbc76R688) and [189](https://github.com/zype/zype-wordpress-plugin/pull/38/files?utf8=%E2%9C%93&diff=split&w=1#diff-7cd6c410c332ccce15ebb07a258fbc76R189)